src/main/java/com/refinedmods/rangedpumps/blockentity/PumpState.java

```diff
@@ -17,7 +18,7 @@ public static Component getMessage(PumpBlockEntity pump) {
             case REDSTONE:
                 return new TranslatableComponent("block.rangedpumps.pump.state.redstone");
             case WORKING:
-                return new TranslatableComponent("block.rangedpumps.pump.state.working", pump.getCurrentPosition().getX(), pump.getCurrentPosition().getY(), pump.getCurrentPosition().getZ(), pump.getRange());
+                return new TranslatableComponent("block.rangedpumps.pump.state.working", pump.getCurrentPosition().getX(), pump.getCurrentPosition().getY(), pump.getCurrentPosition().getZ(), pump.getRange() + " of " + RangedPumps.SERVER_CONFIG.getRange());
```

Comment:
Update the translation to include "of" instead of hardcoding it in here.

Reply:
Fixed